src/frontend/src/pages/ProjectsPage/HCPC.java

```diff
@@ -0,0 +1,5 @@
+public class Main {
```

Comment:
Did you create this file/class?

Reply:
Nope, I didn't create it - only lines 56 to 66

Reply:
Can you remove it ?

Reply:
I just removed it. Seems I had accidentally added the file before.

Reply:
Thanks!